packages/core-modules/contracts/interfaces/IElectionModule.sol

```diff
@@ -13,4 +13,18 @@ interface IElectionModule {
     function nominate() external;
 
     function withdrawNomination() external;
+
+    function setSeatCount(uint seats) external;
+
+    function setEpochDuration(uint duration) external;
+
+    function setPeriodPercent(uint8 percent) external;
```

Comment:
Maybe create a single method for configuring an epoch? So we don't have 3 separate methods removing some noise:

```solidity
function configureCurrentEpoch(uint seats, uint duration, uint nominatePerdiodPercent) external;
```